README-ko.md

```diff
@@ -66,9 +84,9 @@ $ bower install egjs
 ## 의존성
 egjs는 다음의 라이브러리들에 대한 의존성을 가지고 있다.
 
-|[jQuery](https://jquery.com/)|[Hammer.js](http://hammerjs.github.io/)|[Outlayer](https://github.com/metafizzy/outlayer/)|
-|---|---|---|---|
-|1.7.0+ |2.0.4+|1.4.2+|
+|[jQuery](https://jquery.com/)|[Hammer.js](http://hammerjs.github.io/)|
```

Comment:
I think that Document need some description for before version.
`If you use egjs 1.3.0- that you need to outlayer.`